includes/gateway/class-omise-payment-mobilebanking.php

```diff
@@ -0,0 +1,101 @@
+<?php
+defined( 'ABSPATH' ) or die( 'No direct script access allowed.' );
+
+class Omise_Payment_Mobilebanking extends Omise_Payment_Offsite {
+	public function __construct() {
+		parent::__construct();
+
+		$this->id                 = 'omise_mobilebanking';
+		$this->has_fields         = true;
+		$this->method_title       = __( 'Omise Mobile Banking', 'omise' );
+		$this->method_description = wp_kses(
+			__( 'Accept payment through <strong>Mobile Banking</strong> via Omise payment gateway.', 'omise' ),
+			array(
+				'strong' => array()
+			)
+		);
```

Comment:
May be useful to add here that specific mobile banking payment methods will be fetched automatically and only show what the merchant has access to. I'm not really sure how best to phrase it though 😅